### DISCUSSION TITLE
What to do when products are also datasets

### DISCUSSION THREAD

**Discussion by CraigClark** [bcgov/MFIN-Data-Catalogue#145] [General]
@NicoledeGreef 

Something we will need to think about. In the sample data we received, there are 6 data sets. 5 of those are used in product _Home owners grant_, the 6th is the Home owners Grant. This may be a bit confusing to users.

Do you know if products will always be a data-set? Is this an edge case?

**Comment by NicoledeGreef** [comment]
https://bcgov.sharepoint.com/:w:/r/teams/06981/Shared%20Documents/General/architecture/BC-MFIN-Content-Architecture-Living-doc.docx?d=wd7cd5a7fb8454ca29c882a17fe87b27f&csf=1&web=1&e=eNYlcU&nav=eyJjIjoxMDU3OTQ0ODcxfQ%3D%3D

Question for the Business: what is the value in keeping Series and Used in Products?

**Reply by NicoledeGreef** [reply]
Used in Products is being retained.  Two way display:

**Data sets used**
[BCA Data Advice](https://mfin-data-catalogue.apps.silver.devops.gov.bc.ca/data-set/bca-data-advice)
[BCA Folio Address](https://mfin-data-catalogue.apps.silver.devops.gov.bc.ca/data-set/bca-folio-address)
[BCA Folio Description](https://mfin-data-catalogue.apps.silver.devops.gov.bc.ca/data-set/bca-folio-description)
[BCA Values](https://mfin-data-catalogue.apps.silver.devops.gov.bc.ca/data-set/bca-values)

and 

**Used-in data sets**
[Home Owner Grant Dashboard](https://mfin-data-catalogue.apps.silver.devops.gov.bc.ca/data-set/home-owner-grant-dashboard)

Note: bold labels might be changed.

**Reply by NicoledeGreef** [reply]
Keeping **Series** for now. "Series" values comes from a taxonomy which a catalogue admin needs to edit when new values are required.
